index.html

```diff
@@ -91,8 +111,8 @@ <h2 class="card-title">Multi-Post Stories</h2>
     <div class="card-grid">
       <div class="card bg-image3 child-card">
         <div class="card-body">
-          <h2 class="card-title text-white">Profesional Art Printing Data</h2>
-          <p class="card-description text-white">
+          <h2 class="card-title text-white child-title">Profesional Art Printing Data</h2>
```

Comment:
- [ ] please remember, your design should almost be identical to that of the figma version. You are missing 3 cards in both your mobile and desktop design. Please fix that. Consider this a means of practicing to deliver the exact thing a client ask of you. Nice work.

Your design
![Screenshot (1047)](https://user-images.githubusercontent.com/65298048/236174983-82b3a018-8c91-41c7-8b1d-1f0f6d646962.png)
figma design
![Screenshot (1046)](https://user-images.githubusercontent.com/65298048/236175136-b17d32fb-48d6-4ff6-858a-07302e505f2f.png)